pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -87,9 +109,137 @@ func (pl *VolumeBinding) Filter(ctx context.Context, cs *framework.CycleState, p
 	return nil
 }
 
-// New initializes a new plugin with volume binder and returns it.
-func New(_ runtime.Object, fh framework.FrameworkHandle) (framework.Plugin, error) {
+// Reserve is called by the scheduling framework when the scheduler cache is
```

Comment:
Where did you get this from?

`when resources on a node are being reserved for a given Pod`
https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#reserve

But I would prefer that we describe what the function does rather than where it is called.